lib/compress.js

```diff
@@ -1838,6 +1838,11 @@ merge(Compressor.prototype, {
     });
 
     OPT(AST_If, function(self, compressor){
+        if (self.alternative instanceof AST_EmptyStatement) {
+            self.alternative = null;
+            // fall through
+        }
+
```

Comment:
I don't want to disturb the code below it for this trivial fix. Some transforms below it result in empty else clauses. So the other check is not necessarily redundant.

`is_empty()` could be used, but you'll see from the compress tests with no options enabled with `else {}` it does the right thing.